src/chatkit.ts

```diff
@@ -11,17 +11,26 @@ import {
   TokenWithExpiry,
 } from "@pusher/platform-node"
 
-import { getCurrentTimeInSeconds } from "./utils"
+import {
+  getCurrentTimeInSeconds,
+  validateProperties,
+} from "./utils"
 import packageJSON from "../package.json"
+import * as t from 'io-ts'
+import "unknown-ts"
+
+const NonEmptyString = t.refinement(t.string, s => s.length !== 0, "NonEmptyString");
```

Comment:
can pull this in from https://github.com/gcanti/io-ts-types#usage

Reply:
It seems to be more restrictive; it won't take an normal string.
`error TS2322: Type 'string' is not assignable to type 'Branded<string, NonEmptyStringBrand>'.  Type 'string' is not assignable to type 'Brand<NonEmptyStringBrand>'.`

Reply:
ah I see. never mind then